docs/DeveloperGuide.md

```diff
@@ -958,6 +1033,42 @@ testers are expected to do more *exploratory* testing.
 
 1. _{ more test cases …​ }_
 
+### Updating sales of drinks
+
+1. Update the sales of a drink item while all sales are being shown.
+
+    1. Prerequisites: List all sales using the `s-list` command. The full list of drinks sales will be shown.
+    
+    1. Test case: `s-update BSBM/123`
+       
```

Comment:
should there be a blank line between the expected result and the test case?

Reply:
I have placed a blank line to separate the expected result and test case. In my opinion, there is a need for the blank line, but we could discuss this in the meeting to decide.